### PR TITLE
DOC: add parameters and examples to CustomBusinessMonthBegin/End

### PR DESCRIPTION
**Pull request by natmokval**
In the documentation of offsets classes: [CustomBusinessMonthEnd](https://pandas.pydata.org/docs/dev/reference/api/pandas.tseries.offsets.CustomBusinessMonthEnd.html) and [CustomBusinessMonthBegin](https://pandas.pydata.org/docs/dev/reference/api/pandas.tseries.offsets.CustomBusinessMonthBegin.html) there are no parameters, see also and examples sections. I added parameters and examples.

### CODE REVIEW

File: pandas/_libs/tslibs/offsets.pyx [hunk 1]
```diff
@@ -4426,11 +4400,131 @@ cdef class _CustomBusinessMonth(BusinessMixin):
...
+    >>> freq = pd.offsets.CustomBusinessMonthEnd(calendar=bdc)
+    >>> pd.date_range(dt.datetime(2022, 7, 10), dt.datetime(2022, 11, 10), freq=freq)
+    DatetimeIndex(['2022-07-29', '2022-08-31', '2022-09-29', '2022-10-28'],
+                   dtype='datetime64[ns]', freq='CBM')
+    """
+
     _prefix = "CBM"
+    _attributes = tuple(
+        ["n", "normalize", "weekmask", "holidays", "calendar", "offset"]
+    )
```
Comment (MarcoGorelli) [inline pandas/_libs/tslibs/offsets.pyx:4463]: ok with moving the docstring, but I think `_attributes` could have stayed in the parent class? could we move that back please to remove duplication?

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
thanks for working on this! just got a minor comment

**Review by MarcoGorelli — APPROVED** [review]
looks good to me on green, thanks @natmokval

File: pandas/_libs/tslibs/offsets.pyx (hunk 1 above)
Comment (natmokval) [reply pandas/_libs/tslibs/offsets.pyx]: Thank you for reviewing this PR. I moved `_attributes` to `_CustomBusinessMonth`

**Pull request merged**